app/src/main/java/com/example/firstandroid/presentation/add_bathroom/NewBathroomScreen.kt

```diff
@@ -22,31 +22,31 @@ fun AddBathroomDialog(
     onDismissRequest: () -> Unit,
     viewModel: NewBathroomViewModel,
 ) {
-    val state = viewModel.state
+
```

Comment:
You can move all scope work to the ViewModels. Majority of logic should be in VMs.